Sources/Prometheus/MetricTypes/Metric.swift

```diff
@@ -1,34 +1,40 @@
 /// Different types of metrics supported by SwiftPrometheus
 public enum MetricType: String {
-    case counter, gauge, histogram, summary, info
+    /// See `PromCounter`
+    case counter
+    /// See `PromGauge`
+    case gauge
+    /// See `PromHistogram`
+    case histogram
+    /// See `PromSummary`
+    case summary
 }
 
 /// Metric protocol
 ///
 /// See https://prometheus.io/docs/concepts/metric_types/
 public protocol Metric {
```

Comment:
I think this one should follow the Prom prefixing I think, so `PromMetric`

Reply:
Sounds good! 👍